### PR TITLE
 Product Approval Process + Merchant Context

### PR DESCRIPTION
**Pull request by gibmirmilch**
## PR Description

https://spryker.atlassian.net/browse/MP-3934

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:73]: when it's a draft, there is no action buttons, it is waiting for the merchant to send for approval

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:74]: when it's a draft, there is no action buttons, it is waiting for the merchant to send for approval

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:10]: Thus, only the approved, active, within the validity dates range, and existing for the defined store are displayed in the Storefront.

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:26]: and also deny

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:27]: and approve

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:35]: approved, active, within the validity dates range, and existing for the defined store is displayed in the Storefront.

File: docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md
Comment (AdvaBD) [inline docs/marketplace/user/features/202200.0/marketplace-product-approval-process-feature-overview.md:41]: first creating a product and staus is draft

File: docs/marketplace/user/merchant-portal-user-guides/202200.0/products/abstract-products/managing-marketplace-abstract-product.md
Comment (AdvaBD) [inline docs/marketplace/user/merchant-portal-user-guides/202200.0/products/abstract-products/managing-marketplace-abstract-product.md:68]: we will not have for already existing products, only new, and this doc is missing approval process when creating a new product by merchants in the merchant portal

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:72]: Currently, we do have this option in the back office: https://prnt.sc/sEZciJ_gqwqG

Until we have a separate instance of B2B demo shop, I'll leave it as it is, and will change before the release.

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:73]: prevoius comment https://prnt.sc/sEZciJ_gqwqG

File: docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202200.0/catalog/products/managing-products.md:74]: prevoius comment https://prnt.sc/sEZciJ_gqwqG

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:73]: doesn't exist in demo shops

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:83]: doesn't exist in demo shops

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (AdvaBD) [inline docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:87]: doesn't exist in demo shops

**Review by AdvaBD — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:69]: removed this part

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:73]: removed it from the instructions

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:78]: removed

File: docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md
Comment (gibmirmilch) [reply docs/marketplace/user/back-office-user-guides/202204.0/catalog/products/managing-products/managing-products.md:83]: removed

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:13]: please remove "just": https://developers.google.com/style/word-list#letter-j

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:17]: we should introduce procedures with the full sentence: https://developers.google.com/style/procedures

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:20]:
```suggestion
2. From the list of concrete products, select the product you want to create an offer for.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/offers/managing-product-offers.md:59]:
```suggestion
This section describes attributes you see when creating and editing a product offer.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:14]: Please consider revering to "document" and without "just"

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:44]:
```suggestion
3. In the *Stores* pane, in the drop-down menu, select the stores where the product is available.
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md:72]:
```suggestion
For the new product to be available on the Storefront, it needs to be approved. To send the product for approval, do the following:
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:14]: same as above

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:18]:
```suggestion
You can use the following drop-down menus to filter the marketplace concrete products on the *Concrete Products* page:
```

File: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md
Comment (lenadoc) [inline docs/marketplace/user/merchant-portal-user-guides/202204.0/products/concrete-products/managing-marketplace-concrete-product.md:41]: Please revert to "clear": https://developers.google.com/style/word-list#letter-d

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/marketplace/user/features/202204.0/marketplace-product-approval-process-feature-overview.md
Comment (gibmirmilch) [reply docs/marketplace/user/features/202204.0/marketplace-product-approval-process-feature-overview.md:16]: added

**Review by lenadoc — APPROVED** [review]

**Pull request merged**
